Puppet/Puppet/Constants.h

```diff
@@ -2,4 +2,7 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
  */
 
+#import <Foundation/Foundation.h>
+
 static NSString *const kPUPLogTag = @"[Puppet]";
+static NSString *const kMSCustomizedUpdateAlertKey = @"kMSCustomizedUpdateAlertKey";
```

Comment:
That's the puppet so I would suggest not using the `MS` prefix here to better fit a real application.